diff --git a/rootfs/usr/local/bin/startup b/rootfs/usr/local/bin/startup
index b8ee399..55a8882 100644
--- a/rootfs/usr/local/bin/startup
+++ b/rootfs/usr/local/bin/startup
@@ -23,12 +23,11 @@ sed -i "s/<OPCACHE_MEMORY_LIMIT>/${OPCACHE_MEMORY_LIMIT}/g" /etc/php8/conf.d/00_
 # Set permissions for /flarum folder
 echo "[INFO] Setting folder permissions"
 for folder in /etc/s6.d /run/php /var/log /var/lib/nginx; do
-  find ${folder} ! -user "${UID}" -exec chown -h "${UID}:${GID}" {} \+
-  find ${folder} ! -group "${GID}" -exec chown -h "${UID}:${GID}" {} \+
+  echo {$folder}
+  chown -R "${UID}:${GID}" ${folder}
 done
 
-find /flarum ! -user "${UID}" -exec chown "${UID}:${GID}" {} \+
-find /flarum ! -group "${GID}" -exec chown "${UID}:${GID}" {} \+
+chown -R "${UID}:${GID}" /flarum
 
 # Set log output to STDOUT if wanted (LOG_TO_STDOUT=true)
 if [ "${LOG_TO_STDOUT}" = true ]; then
